share/templates.d/99-generic/runtime-cleanup.tmpl

```diff
@@ -363,6 +363,9 @@ removepkg cdparanoia-libs opus libtheora libvisual flac-libs gsm avahi-glib avah
 ## metacity requires libvorbis and libvorbisfile, but enc/dec are no longer needed
 removefrom libvorbis --allbut /usr/${libdir}/libvorbisfile.* /usr/${libdir}/libvorbis.*
 
+## Remove build-id links, they are used with debuginfo
+remove /usr/${libdir}/.build-id
```

Comment:
${libdir} needs to be just plain old /lib/

Reply:
I'll fix and rebase